TileStache/Goodies/VecTiles/transform.py

```diff
@@ -345,20 +346,16 @@ def road_sort_key(shape, properties, fid, zoom):
         sort_val -= 1
 
     if zoom >= 15:
-        # Bridges and tunnels add +/- 10
         bridge = properties.get('bridge')
         tunnel = properties.get('tunnel')
         if bridge in ('yes', 'true'):
-            sort_val += 10
+            sort_val += 40
         elif (tunnel in ('yes', 'true') or
               (railway == 'subway' and tunnel not in ('no', 'false'))):
             sort_val -= 10
 
-        # Keep aerialways above (almost) everything else, including bridges,
-        # but make sure it doesn't go beyond the 0-34 range. (still need to
-        # leave space for explicit layer).
-        if aerialway != '':
-            sort_val = min(34, sort_val + 10)
+        if aerialway:
+            sort_val += 30
```

Comment:
Looking at TagInfo, only a couple dozen aerialway features have tunnel or bridge, but according to  the OSM wiki, 25% of roads with `bridge=yes` aren't tagged with a layer. To account for this we should +40 aerialway features (the same amount as for bridges) to keep aerialway always above roads. _This could be consolidated with the bridge section above?_

(The later section for `layer` order will force it to the correct data values when an OSM editor has taken the time to distinguish which way goes in what order.)